examples/connection_tester.cpp

```diff
@@ -484,7 +484,7 @@ struct peer_conn
 		unsigned int length = read_uint32(ptr);
 		if (length > sizeof(buffer))
 		{
-			std::fprintf(stderr, "len: %d\n", length);
+			std::fprintf(stderr, "len: %u\n", length);
```

Comment:
I'm surprised GCC didn't warn about this

Reply:
Perhaps warning is disabled or GCC version is too low. Streams doesn't have such potential problem